### PR TITLE
Inkscape 1.0 compatibility

### PR DESCRIPTION
**Pull request by aaujon**
Fix Inkscape 1.0 compatibility, for the record changes are explained here : https://wiki.inkscape.org/wiki/index.php/Updating_your_Extension_for_1.0
Tested with 1.0 beta 2

### CODE REVIEW

**Comment by aaujon** [comment]
should fix #216

**Comment by frmdstryr** [comment]
Thanks for doing this this. 

Can it also be made backwards compatible so that it works with both versions of inkscape for those running older versions?

**Comment by aaujon** [comment]
Unfortunatly some changes are not backwarding compatible (inkex.Effect.selected > inkex.Effect.svg.selected) for exemple. I guess we'll have to maintain 2 versions, which isn't that bad since this code doesn't need much updates.

**Comment by frmdstryr** [comment]
Does inkex have any kind of version number that could be checked?

Ex:
```python
nodes = self.selected if inkex.version < 1.0 else self.svg.selected
```

**Comment by aaujon** [comment]
I did a quick search and looks like it's not possible, the 'official' extensions made by inkscape folks are now in a different git are not compatible with < 1.0.
I didn't find a third party extension that kept backward compatibility for the moment. I think we can support both in 2 different directories.

**Comment by aaujon** [comment]
I did another quick search I can't find a nice way to support both inkscape version in the code, maintaining 2 versions seems the only way for some time. If it's okay with you I can  update the PR to add another folder inkscape1.0/

**Comment by frmdstryr** [comment]
Since there's not too many changes I think simply using a few hasattr checks would be enough to determine if it's 1.0 or not.

**Comment by aaujon** [comment]
Ok, I think that should be good now but I couldn't check with inkscape 0.9

File: plugins/inkscape/inkcut.py [hunk 1]
```diff
@@ -23,10 +23,19 @@
...
 import os
 import inkex
+from lxml import etree
 from subprocess import Popen, PIPE
 from shutil import copy2
 from distutils.spawn import find_executable
 
+# check inkscape version
+import importlib
+optparse_spec = importlib.util.find_spec("optparse")
```
Comment (aaujon) [inline plugins/inkscape/inkcut.py:33]: optparse is only present in inkscape 1.0

**Comment by frmdstryr** [comment]
Thank you!

**Pull request merged**